main.js

```diff
@@ -1 +1,28 @@
 'use strict';
+
+let treeUl = document.getElementsByTagName('ul')[0];
+let treeLis = treeUl.getElementsByTagName('li');
+
+for (let i = 0; i < treeLis.length; i++) {
+  let li = treeLis[i];
+  let span = document.createElement('span');
+  
+  li.insertBefore(span, li.firstChild);
+  span.appendChild(span.nextSibling);
+}
+
+let tree = document.getElementsByTagName('ul')[0];
+
+tree.onclick = function(event) {
+  let target = event.target;
+
+  if (target.tagName != 'SPAN') {
+    return; // клик был не на заголовке
```

Comment:
do need comments like this in production code. Also, comment same as other code should be in English